### ISSUE TITLE
Confusingly structured Installation section

### ISSUE THREAD

**Issue by giorgim** [facebook/react#9275]
Hey guys, I want to bring out some issue in docs, Install section. https://facebook.github.io/react/docs/installation.html. This happened to me and one of my colleagues and I guess others may get confused too.

For example I want to use create-react-app approach for installing react.
This means I need to follow ONLY this section right: https://facebook.github.io/react/docs/installation.html#creating-a-single-page-application

But after my colleague followed that, he then thought had do follow things like this too:
https://facebook.github.io/react/docs/installation.html#enabling-es6-and-jsx

In other words the installation docs, doesn't say that if you follow the first link I gave for example above, following second link is not necessary as create-react-app includes Babel for example.
Isn't it?

I think the doc has to mention that it is an OR approach, either use this section, or this, etc.

**Comment by gaearon** [comment]
Ooh. Thanks for raising this, I also saw people trying to add Babel on top and was very confused why they do this. This explains it.

Would you like to send a PR to make it clearer?

**Comment by gaearon** [comment]
Ideally I think we should make a tabular installation page, like in React Native docs. So that the user chooses one way at the top and then the rest of the instructions are only specific to that.

**Comment by giorgim** [comment]
@gaearon Dear Dan please feel free to add that improvement yourself. Indeed docs should make it clear, what path to take. @gaearon Also I didn't send a PR with previous issue with not using Keys in react if you remember, I also invite you (or someone else) to improve that. It's fine for me to know that these "opened" issues have helped already. I will delegate rest to people who know more about subject matter.

**Comment by filiphosko** [comment]
I created a PR for this https://github.com/facebook/react/pull/9277 , just wanted to help until the final solution will be done.
